### PR TITLE
Anna Kondratiuk

### PR DESCRIPTION
**Pull request by annakondr**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        return this.numbers
+    }
+
+    removeNumber(n) {
+        return this.numbers.filter(num => num !== n)
+    }
+
+    sum() {
+        if (this.numbers.length !== 0) {
+            return this.numbers.reduce((a, b) => {return a + b})
```
Comment (gestap) [inline src/main.js]: no need for curly braces
https://github.com/mate-academy/style-guides/blob/master/javascript.md#arrows--implicit-return

File: src/main.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        }
+        return 0;
+    }
+
+    product() {
+        if (this.numbers.length !== 0) {
+            return this.numbers.reduce((a, b) => {return a * b})
```
Comment (gestap) [inline src/main.js]: here too

File: src/main.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        }
+        return 1;
+    }
+
+    min() {
+        if (this.numbers.length !== 0) {
+            return Math.min.apply(null, this.numbers)
+        }
+        return ('Error: The Numberdrome is empty')
```
Comment (gestap) [inline src/main.js]: you need to use `throw`
`... or throws an error ...`

File: src/main.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    }
+
+    max() {
+        if (this.numbers.length !== 0) {
+            return Math.max.apply(null, this.numbers)
+        }
+        return ('Error: The Numberdrome is empty')
```
Comment (gestap) [inline src/main.js]: here too

File: src/main.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    min() {
+        if (this.numbers.length !== 0) {
+            return Math.min.apply(null, this.numbers)
+        }
+        return ('Error: The Numberdrome is empty')
+    }
+
+    max() {
+        if (this.numbers.length !== 0) {
+            return Math.max.apply(null, this.numbers)
```
Comment (gestap) [inline src/main.js]: suggestion: you can use spread operator

File: src/main.js [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+    product() {
+        if (this.numbers.length !== 0) {
+            return this.numbers.reduce((a, b) => {return a * b})
+        }
+        return 1;
+    }
+
+    min() {
+        if (this.numbers.length !== 0) {
+            return Math.min.apply(null, this.numbers)
```
Comment (gestap) [inline src/main.js]: suggestion: you can use spread operator

**Review by ost-ptk — CHANGES_REQUESTED** [review]

**Review by ost-ptk — APPROVED** [review]

**Pull request closed without merging**